web_poet/example.py

```diff
@@ -0,0 +1,86 @@
+from asyncio import run
```

Comment:
Could you please add a docstring to this module? I think we should state here that it's a toy example for the purpose of making it easier for the tutorial, and discourage people from using it in production or relying on the API in this module. What do you think?

Reply:
Added a warning message on import as well.